ga4gh/cli.py

```diff
@@ -196,6 +202,9 @@ def addGlobalOptions(parser):
     parser.add_argument(
         "--config-file", "-F", type=str,
         help="The configuration file to use")
+    parser.add_argument(
```

Comment:
When starting the server within a test, we want one instance of the server to be launched, not two processes.  If the later, it makes it difficult to kill the server.

(Reloading is only useful to pick up changes in code when developing; when running a test this should not be happening.)